src/components/Exchange.js

```diff
@@ -1456,25 +1496,88 @@ export default class Exchange extends React.Component {
+                      console.log(unspent, config, color);
+                      console.log(amount);
+                      tx = Tx.transferFromUtxos(
+                        unspent, this.state.daiAddress,
+                        config.exitHandlerAddr, amount, color,
+                      );
+                      return Promise.all([
+                        tx.signWeb3(this.props.web3),
+                        this.state.xdaiweb3.eth.getTransaction(
+                          bufferToHex(tx.inputs[0].prevout.hash)
+                        ),
+                      ]);
+                    })
+                    .then(([signedTx, _inputTx]) => {
+                      console.log(inputTx);
+                      inputTx = _inputTx;
+                      console.log('TxHash:', signedTx.hash());
+                      return this.state.xdaiweb3.eth.sendSignedTransaction(
+                        signedTx.hex()
+                      );
+                    })
+                    .then(transferTx => {
+                      const tx = Tx.fromRaw(transferTx.raw);
+                      
+                      const utxoId = (new Outpoint(tx.hash(), 0)).getUtxoId();
+                      const sigHashBuff = Exit.sigHashBuff(utxoId, bi(amount));
+                      const sigHash = `0x${sigHashBuff.toString('hex')}`;
+                      setup = {
+                        inputTx,
+                        tx: transferTx,
+                        effectiveBlock: periodBlockRange(transferTx.blockNumber)[1],
+                        sigHashBuff,
+                        sigHash,
+                      };
+
+                      return Tx.signMessageWithWeb3(this.props.web3, sigHash);
+                    })
+                    .then(sig => {
+                      const vBuff = Buffer.alloc(32);
+                      vBuff.writeInt8(sig.v, 31);    
+                      const signedData = Exit.bufferToBytes32Array(
+                        Buffer.concat([setup.sigHashBuff, Buffer.from(sig.r), Buffer.from(sig.s), vBuff])
+                      );
+                      delete setup.sigHash;
+                      delete setup.sigHashBuff;
+                      setup.signedData = signedData;
+                      console.log(setup);
+                      return fetch(
+                        'https://2nuxsb25he.execute-api.eu-west-1.amazonaws.com/testnet/sellExit',
+                        {
+                          method: "POST",
+                          mode: "cors",
+                          cache: "no-cache",
+                          headers: {
+                              "Content-Type": "application/json",
+                          },
+                          body: JSON.stringify(setup),
+                        }
+                      );
                     })
+                    .then(response => response.json())
+                    .then(rsp => {
+                      console.log(rsp);
```

Comment:
This closes the exchange dialogue after successfully exiting.

```suggestion
                      this.setState({amount:"",daiToXdaiMode:false})
                      console.log(rsp);
```